pipeline/src/pipeline.rs

```diff
@@ -487,24 +487,11 @@ impl<T: FieldElement> Pipeline<T> {
         Ok(())
     }
 
-    fn maybe_write_constants(&self, constants: &[(String, Vec<T>)]) -> Result<(), Vec<String>> {
-        if let Some(path) = self.path_if_should_write(|name| format!("{name}_constants.bin"))? {
```

Comment:
where are these names given now?

Reply:
They are not. The only thing that differentiates one output from another is the output dir.